packages/valory/skills/chatui_abci/handlers.py

```diff
@@ -84,15 +84,28 @@ class HttpMethod(Enum):
 
 
 # Content type constants
-DEFAULT_HEADER = HTML_HEADER = "Content-Type: text/html\n"
-CONTENT_TYPES = {
-    ".js": "Content-Type: application/javascript\n",
-    ".html": HTML_HEADER,
-    ".json": "Content-Type: application/json\n",
-    ".css": "Content-Type: text/css\n",
-    ".png": "Content-Type: image/png\n",
-    ".jpg": "Content-Type: image/jpeg\n",
-    ".jpeg": "Content-Type: image/jpeg\n",
+class HttpContentType(Enum):
+    """Enum for HTTP content types."""
+
+    HTML = "Content-Type: text/html\n"
+    JS = "Content-Type: application/javascript\n"
+    JSON = "Content-Type: application/json\n"
+    CSS = "Content-Type: text/css\n"
+    PNG = "Content-Type: image/png\n"
+    JPG = "Content-Type: image/jpeg\n"
+    JPEG = "Content-Type: image/jpeg\n"
```

Comment:
We could avoid some duplication.

```suggestion
class HttpContentType(Enum):
    """Enum for HTTP content types."""
    HTML = "text/html"
    JS = "application/javascript"
    JSON = "application/json"
    CSS = "text/css"
    PNG = "image/png"
    JPG = "image/jpeg"
    JPEG = "image/jpeg"

    @property
    def header(self) -> str:
        return f"Content-Type: {self.value}\n"
```

Then, instead of `value`, we could use `header`.

Reply:
e1c2674b4d7f5319602b25e0e2a8f3a0ab4a7d4c